tests/yakus/afterkan.test.cpp

```diff
@@ -0,0 +1,69 @@
+#include <gtest/gtest.h>
+#include "hands.h"
+#include "hand.h"
+#include "gamestate.h"
+#include "handformer.h"
+#include "handnode.h"
+#include "analysis.h"
+#include "statefunctions.h"
+
+using Mahjong::GameState;
+using Mahjong::Hand;
+using Mahjong::HandFromNotation;
+using Mahjong::breakdownHand;
+using Mahjong::Node;
+using Mahjong::Piece;
+
+TEST(isAfterAKan, 1Han) {
+  auto gameState = GameState();
+  gameState.hands[0] = Hand(HandFromNotation("123m789m1111z999s55z"));
+  gameState.currentPlayer = 0;
+  
+  gameState.prevState = Mahjong::Replacement;
+
+  auto root = breakdownHand(gameState.hands.at(0).live);
+
+  for (const auto& branch : Node::AsBranchVectors(root.get())) {
```

Comment:
The node class should be tested at some point as well but maybe wait til later on it.

Reply:
Would be a good PR, will add to the trello